### ISSUE TITLE
Option to re-scale the size of markers for contact, force, perturbations, etc...

### ISSUE THREAD

**Issue by PeterMitrano** [deepmind/mujoco#912]
Hi,

I'm a phd student using the Panda robot model from the Menagerie. I've noticed that for this and most other robot models I've tried, the scale of the visualization is really large, and obstructs the view of what's in contact, for example. The screenshot below shows that I mean (the two robot are holding tools, but you can hardly see anything with the giant markers).

Is there any way to adjust this without compiling from source? Could we expose a parameter for this in the viewer somewhere, like with font scale?

![image](https://github.com/deepmind/mujoco/assets/4010770/bbe42208-1a25-44af-bc44-375bb57855e3)

**Comment by v-r-a** [comment]
Hi, I'm a MuJoCo user.

See: 
https://mujoco.readthedocs.io/en/latest/XMLreference.html#visual-scale

You can set this by adding the following to the XML (panda.xml):

```
<visual>
       <scale forcewidth = "0.1"  contactwidth = "0.1" contactheight = "0.1" />
 </visual>
```

**Comment by PeterMitrano** [comment]
Thank you!

**Comment by yuvaltassa** [comment]
btw, the latest version of the viewer lets you adjust these values in the GUI. Until the next release you'll need to build it from source.

**Comment by yuvaltassa** [comment]
Ah there's even a screencast 🙂 
https://youtu.be/mXVPbppGk5I

**Comment by happygaoxiao** [comment]
Hello, I watched your video. But I cannot have the option of `Visualization` below `Render`, with MuJoCo 2.3.5.
![image](https://github.com/deepmind/mujoco/assets/24514908/e9e74576-2cf2-48af-b022-d0c33a4883f0)

By the way, there is a function to add markers in [mujoco-py](https://github.com/openai/mujoco-py/blob/a13903b82f1ab316815e63b3575526005b3b2ae1/mujoco_py/mjrendercontext.pyx#L249) . We can set several types of markers. Is there any similar functions to do this? I cannot find any options.

Here is an example where I did with `mujoco-py`. The goal is to visualize all contact points during the hand exporation.
![image](https://github.com/deepmind/mujoco/assets/24514908/93f02a57-8f43-48b6-8228-8f887d30781f)

**Comment by yuvaltassa** [comment]
This was released in MuJoCo 2.3.6
